app/views/components/_notice.html.erb

```diff
@@ -1,14 +1,20 @@
 <% if defined?(title) %>
   <%
-    description_text ||= ""
-    description_govspeak ||= ""
+    description_text ||= false
+    description_govspeak ||= false
   %>
   <section class="app-c-notice" aria-label="Notice">
-    <h2 class="app-c-notice__title"><%= title %></h2>
-    <% if description_text.present? %>
+    <% if description_text || description_govspeak %>
+      <h2 class="app-c-notice__title"><%= title %></h2>
+    <% else %>
+      <p class="app-c-notice__title"><%= title %></p>
```

Comment:
Nit: Should this be a `span`? Since we want it to have no semantic meaning?

(Also less likely to be impacted by anyone doing pesky p globals 😱 )